sass/blocks/footer.sass

```diff
@@ -0,0 +1,17 @@
+// Footer
+
+.footer-block
+  padding: 2rem
+  background-color: $lightHappyBlue
+  border-radius: 0px 0px 5px 5px
```

Comment:
Really like that you thought ahead and used browser prefixes! Maybe a case to create a @mixin?